thoth/common/openshift.py

```diff
@@ -1579,6 +1583,16 @@ def schedule_adviser(
             runtime_environment
         )
 
+        template_parameters["THOTH_ADVISER_METADATA"] = json.dumps(
```

Comment:
This change should also land in `run_adviser`. Otherwise, parameters propagated to `run_adviser` method will not match to parameters provided by `schedule_adviser` method.